### ISSUE TITLE
Add CTRL+BACKSPACE to default keymap for backward word delete

### ISSUE THREAD

**Issue by cowuake** [jcubic/jquery.terminal#920]
#### Premise
As of today, it seems that the user is not able to (backward) delete a word on common desktop browsers (Firefox, Chrome, Edge), which take priority over keyboard events and prevent using, e.g., the CTRL+W shorcut.

#### Proposal
Since CTRL+BACKSPACE is an even more common shortcut for the same purpose and usually available on CLIs, here is made the proposal to include it in the default keymap. I can open a PR myself, including [this change](https://github.com/jcubic/jquery.terminal/compare/devel...cowuake:jquery.terminal:devel).

#### Expected impact
Since there is no remapping involved, but only the addition of a new shortcut made available by default, the impact should be minimal and limited to a fairly improved comfort for the generic user.

**Comment by jcubic** [comment]
Good idea, this is not part of Bash, but it's default in Emacs (a lot of shortcuts in Bash came from Emacs).

**Comment by jcubic** [comment]
Please don't close, I keep the issue open until it land on master and is released. That's why I added a label resoved.
